<?xml version="1.0"?>

<launch>
  <arg name="uri" default="radio://0/80/2M/E7E7E7E704" />

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>

  <group ns="cf1">
    <node pkg="crazyflie_driver" type="crazyflie_add" name="crazyflie_add" output="screen">
      <param name="uri" value="$(arg uri)" />
      <param name="tf_prefix" value="cf1" />

      <param name="enable_logging" value="True" />
      <param name="use_ros_time" value="True" />
      <param name="enable_logging_imu" value="True" />
      <param name="enable_logging_temperature" value="True" />
      <param name="enable_logging_magnetic_field" value="True" />
      <param name="enable_logging_pressure" value="True" />
      <param name="enable_logging_battery" value="True" />
      <param name="enable_logging_pose" value="True" />
      <param name="enable_logging_packets" value="True" />
    </node>

  </group>

  <node name="hover_controller" pkg="swarm_manager" type="Hover.py" output="screen">
  </node>

</launch>
